tests/integration/test_proteus_start.py

```diff
@@ -1,29 +0,0 @@
-from __future__ import annotations
-
-import os
-from pathlib import Path
-
-import pytest
-from helpers import PROTEUS_ROOT
-
-from proteus import Proteus
-
-if os.getenv('CI'):
-    # https://github.com/FormingWorlds/PROTEUS/pull/149
-    pytest.skip(reason='No way of currently testing this on the CI.', allow_module_level=True)
-
-
-def test_dummy_run():
-    config_path = PROTEUS_ROOT / 'input' / 'dummy.toml'
-
-    runner = Proteus(config_path=config_path)
-
-    runner.config['log_level'] = 'WARNING'
-    runner.config['plot_iterfreq'] = 0
-    runner.config['iter_max'] = 0
```

Comment:
Consider playing with the number of iterations to speed up the test. On my machine it takes about 13 minutes to run the test, if I set iter_max to 0 it goes to about 22 seconds. 

Even iter_max of 10 or 100 is quite manageable, as it only adds a few seconds of runtime. However, plot_iterfreq is a permormance killer, considering that the default in dummy.toml is 1 (plot every iteration). Consider setting to 0 to only plot at the end.

| iter_max | plot_iterfreq | time (s) |
|-|-|-|
| 0 | 0 | 22 |
| 10 | 0 | 23 |
| 100 | 0 | 28 |
| 10 | 1 | 49 |


iter_max=10 and

Reply:
Set plot_iterfreq to zero.